utilities/shared.mjs

```diff
@@ -1,5 +1,10 @@
 
-// This function is an internal util, but we may want to test it in tests so we export it here for now.
+
+/**
+ * Check if the supplied input is valid JSON or not.
+ * @param input A string or Object that should be JSON conformant.
+ * @return boolean For whether or not the supplied input was JSON conformant.
+ */ 
 export function isValidJSON(input=""){
    if(input){
       try{
```

Comment:
Not sure what's happening here. You are preempting some checks with the default "" set. with a try here, most things are all caught...

```js
function isValidJSON(input) {
  try {
    return Boolean((typeof input === "string") ? JSON.parse(input) : JSON.parse(JSON.stringify(input)))
  } catch(err) {
    return false
  }
}
```

Maybe. I don't know

Reply:
String or object comes in.  If it can be parsed to JSON, return true.  Otherwise, return false.  Make sure "" and null values return false.  That's what it does.